src/components/Modal/style.css

```diff
@@ -32,31 +32,39 @@
 }
 
 .inner {
+  /*
+   * We set the `font-size` to 16px here. This will fix the rest of the SDK
+   * (which uses relative units) to this absolute unit.
+   *
+   * To opt-in to relative font-sizes, you can override this style to a relative
+   * font-size e.g. `font-size: 1em`
+   */
+  font-size: 16px;
```

Comment:
Is my understanding right that the root is now the only place where px is used? All the other usage of px will be replaced with `unit` calculation?

Reply:
Yes, that's the plan
We _can_ have `px` used in the SDK, but it should be reserved for things we really don't want to be scaled for some reason